pkg/cmd/completion.go

```diff
@@ -72,6 +72,11 @@ var completionInstallCmd = &cobra.Command{
 		var buf bytes.Buffer
 		var path string
 		var name string
+
+		if utils.IsWindows() {
+			utils.HandleError(fmt.Errorf("Completion files are not supported on windows currently. However you can use completion files with Windows Subsystem for Linux."))
```

Comment:
nit: you can use `errors.New` here since we don't need a template string

```suggestion
			utils.HandleError(errors.New("Completion files are not supported on windows currently. However you can use completion files with Windows Subsystem for Linux."))
```

Reply:
Resolved by 25e04da